inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -159,6 +159,23 @@ public void registerKnowledgeBase(KnowledgeBase kb, RepositoryImplConfig cfg)
         repoManager.addRepositoryConfig(new RepositoryConfig(repositoryId, cfg));
         entityManager.persist(kb);
     }
+    
+    @Override
+    public void defineBaseProperties(KnowledgeBase kb, boolean kbUpdateFlag) 
```

Comment:
`aKb`

Parameter `kbUpdateFlag` is not used.

Reply:
Its for letting the function aware of the updating the KB or not. Can be removed for now, but we would definitely need that when we implement a different approach for updating the KB.